lib/core/test/unit/Cardano/Pool/MetricsSpec.hs

```diff
@@ -429,7 +433,7 @@ instance Arbitrary PoolRegistrationCertificate where
     arbitrary = PoolRegistrationCertificate
         <$> arbitrary
         <*> fmap (L.nub . getNonEmpty) (scale (`mod` 3) arbitrary)
-        <*> fmap toEnum (choose (0, 100))
+        <*> ((1 %) <$> choose (1, 10000))
```

Comment:
I think this instance is fine, particularly since it seems unimportant to the tests in this module.